src/app/screens/LNURLPay.tsx

```diff
@@ -58,7 +58,7 @@ function LNURLPay(props: Props) {
         lnurl.getDetails(lnurlString).then((lnurlDetails) => {
           if (lnurlDetails.tag === "payRequest") {
             setDetails(lnurlDetails);
-            setValueMSat(lnurlDetails.minSendable);
+            setValueSat((lnurlDetails.minSendable / 1000).toString());
```

Comment:
is `lnurlDetails.minSendable` always defined? what if the response does not include it?

Reply:
Currently in our interface we haven't defined them as optional, as the spec doesn't speak about this as far as I'm aware? https://github.com/fiatjaf/lnurl-rfc/blob/luds/06.md

Reply:
ok.thx